doc/WELOTEC.md

```diff
@@ -18,4 +18,6 @@
 | **OTA Update**                       | yes             |
 | **Factory Reset**                    | yes             |
 | **Wifi Commissioning via Bluetooth** | no              |
-| **LTE Support**                      | experimental    |
+| **LTE Support** [^1]                 | experimental    |
+
+[^1]: in [LTE documentation](LTE.md) you can find more details concerning configuration
```

Comment:
do you have a idea why a link symbol at the end of the line is present?

Reply:
see: https://github.com/HarryWaschkeit/meta-omnect/blob/hwt-lte-default-config-and-docu/doc/WELOTEC.md#user-content-fnref-1-bca1608a12dcceb80942b7ef9e0fef1d

Reply:
this is standard markdown behavior (at least of github) for footnotes, maybe to have a perma-link also at hand in the footnote itself.